integration-libs/opf/components/opf-checkout-payment-and-review/opf-checkout-payment-and-review.component.spec.ts

```diff
@@ -66,18 +94,26 @@ describe('OPFCheckoutPaymentReviewComponent', () => {
     cart$.next(mockCart);
     fixture.detectChanges();
 
-    expect(
-      fixture.debugElement.query(By.css('cx-opf-checkout-payments'))
-    ).toBeTruthy();
+    expect(el.query(By.css('cx-opf-checkout-payments'))).toBeTruthy();
   });
 
   it('should not render cx-opf-checkout-payments component if payment type is set to ACCOUNT', () => {
     cart$.next({ ...mockCart, paymentType: { code: 'ACCOUNT' } });
 
     fixture.detectChanges();
 
-    expect(
-      fixture.debugElement.query(By.css('cx-opf-checkout-payments'))
-    ).toBeFalsy();
+    expect(el.query(By.css('cx-opf-checkout-payments'))).toBeFalsy();
+  });
+
+  it('should change form value when checkbox get selected / change state', () => {
```

Comment:
This test seems to be unstable because it's failing occasionally (3 times for 10 runs on local env). Worth to investigate.
![image](https://user-images.githubusercontent.com/28891782/214256797-2cad1eb3-b088-400d-aa8b-44c3e9d777b1.png)